apps/sms/views/conversation/js/conversation.js

```diff
@@ -1933,6 +1946,18 @@ var ConversationView = {
     new OptionMenu(params).show();
   },
 
+  addRecipients: function() {
+    var draft = new Draft({
+      recipients: this.activeThread.participants,
+      content: Compose.getContent(),
+      subject: Compose.getSubject(),
+      type: Compose.getType()
+    });
+    this.initiateNewMessage({ draft });
```

Comment:
nit: here is a slightly different proposal, tell me what you think:

```javascript
var content = {
  recipients: this.activeThread.participants,
  content: Compose.getContent(),
  subject: Compose.getSubject(),
  type: Compose.getType()
};
this.initiateNewMessage({ content });
```

and in initiateNewMessage:
```javascript
  ... if (parameters.content) {
    var draft = new Draft(content);
    draftCreatePromise = Drafts.add(draft).store().then(() => draft.id);
  }
```

so quite similar but I think it's slightly simpler like this. We don't need to use "Draft" at all in `addRecipients` for example.